docs-site/docs/docs/about-menu.md

```diff
@@ -0,0 +1,43 @@
+The about menu provides a deeper view into your instance of Polynote. 
+
+![Settings Menu](images/settings-menu.png)
+
+The left pane contains several sections to view different pieces of data about Polynote. 
+
+### About 
+The about menu contains information about your instance's version and most recent server commit. 
+
+### Hotkeys 
+The hotkeys menu contains a list of all supported keyboard shortcuts in Polynote. 
+
+### Preferences 
+The preferences menu contains two sections - _UI preferences_ and _storage_.
+
+#### UI Preferences 
+The preferences menu is where you can customize the following settings: 
```

Comment:
Might be worth commenting that these preferences are stored in the browser - using another browser / clearing storage will clear them. (there's also a Clear All button too)